src/cgnstypes.h.in

```diff
@@ -36,6 +36,7 @@
 #define CG_BUILD_BASESCOPE @BUILDBASESCOPE@
 #define CG_BUILD_PARALLEL  @BUILDPARALLEL@
 #define CG_BUILD_COMPLEX_C99_EXT @HAVE_COMPLEX_C99_EXT@
+#define CG_BUILD_HDF5   @BUILDHDF5@
```

Comment:
This line is not needed anywhere.
I am not in favor of adding this define as it will push users to have different behavior between HDF5 and ADF

Reply:
I made a mistake, this was already defined earlier, so I removed the line. Do you mean the addition to cgnstype_f03.h.in?
I added it to Fortran since it was defined for C, and I was using it in a Fortran test. But I can just use cg_set_file_type_f to determine if HDF5 was enabled instead.

Reply:
Ok nevermind